lib/fs_utils/source_file.js

```diff
@@ -114,7 +115,8 @@ const makeCompiler = (path, cache, linters, compilers, wrap) => {
 class SourceFile {
   constructor(path, compilers, linters, wrapper, isHelper, isVendor, fileList) {
     this.fileList = fileList;
-    const type = compilers && compilers[0] && compilers[0].type;
+    // treat json files from node_modules as javascript
+    const type = compilers && compilers[0] && compilers[0].type || deppack.isNpm(path) && 'javascript';
```

Comment:
not the best piece but then, not sure if there is a better way to let deppack influence file type...